l10n/fr.js

```diff
@@ -96,6 +96,8 @@ OC.L10N.register(
     "Shared boards" : "Tableaux partagés",
     "View more" : "Voir plus",
     "Move board to archive" : "Déplacer le tableau vers l'archive",
-    "Create a new board" : "Créer un nouveau tableau"
+    "Create a new board" : "Créer un nouveau tableau",
```

Comment:
Please remove the changes to the translation files, those are added by transifex.

Reply:
fixed in 02926d93cb1864e16c4c6d628af4afdadb17bf5b